test/guest/ipa-fedora-22.setup

```diff
@@ -10,18 +10,9 @@ iptables -F
 dnf -y --enablerepo=updates-testing update
 dnf -y --enablerepo=updates-testing install freeipa-server bind bind-dyndb-ldap
 
-hostnamectl set-hostname f0.cockpit.lan
+# bug in selinux-policy-targeted-3.13.1-128.13.fc22
+# HACK https://bugzilla.redhat.com/show_bug.cgi?id=1224211
+# we need to restart systemctl and then reboot the system
+systemctl daemon-reexec
 
-# Let's make sure that ipa-server-install doesn't block on
-# /dev/random.
-#
-rm /dev/random
-ln -s /dev/urandom /dev/random
-
-ipa-server-install -U -p foobarfoo -a foobarfoo -n cockpit.lan -r COCKPIT.LAN --setup-dns --no-forwarders
-
-# Make sure any initial password change is overridden
-printf 'foobarfoo\nfoobarfoo\nfoobarfoo\n' | kinit admin@COCKPIT.LAN
-
-ln -sf ../selinux/config /etc/sysconfig/selinux
-echo 'SELINUX=permissive' > /etc/selinux/config
+dnf clean all
```

Comment:
Why is this line after systemctl daemon-rexec in one setup, and before it in another?